### PR TITLE
fix(services): move data files from feature module to `app/data`

### PR DESCRIPTION
**Pull request by ViktorSlavov**
Move all data files from `services` feature module to `app/data` folder.

In the future, we would add any new existing data file to the data root, as this would lead to more reusability and minimum redundancy

### CODE REVIEW

File: live-editing/configs/ExportExcelConfigGenerator.ts [hunk 1]
```diff
@@ -24,33 +24,33 @@ export class ExportExcelConfigGenerator implements IConfigGenerator {
 
         configs.push(new Config({
             component: 'ExcelExportSample1Component',
-            additionalFiles: ["/src/app/services/export-excel/data/invoiceData.ts"],
+            additionalFiles: ['/src/app/data/invoiceData.ts'],
```
Comment (gedinakova) [inline live-editing/configs/ExportExcelConfigGenerator.ts:27]: Grid Excel Export sample is broken in stackblitz/codesandbox - not finding the data.

File: src/app/services/localization-samples/localization-all-resources/localization-all-resources.component.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import localeJA from '@angular/common/locales/ja';
 import localeKO from '@angular/common/locales/ko';
 import localeHans from '@angular/common/locales/zh-Hans';
 import localeHant from '@angular/common/locales/zh-Hant';
-import { DATA } from '../../data/nwindData';
+import { DATA } from '../../../data/nwindData';
```
Comment (gedinakova) [inline src/app/services/localization-samples/localization-all-resources/localization-all-resources.component.ts:16]: This sample is also failing in stackblitz/codesandbox for missing data.

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: live-editing/configs/ExportExcelConfigGenerator.ts (hunk 1 above)
Comment (ViktorSlavov) [reply live-editing/configs/ExportExcelConfigGenerator.ts:27]: Thanks for catching that! I forgot that `shortenComponentPath` interferes w/ the import path on live editing. They should now be working properly

**Review by gedinakova — APPROVED** [review]

**Comment by ViktorSlavov** [comment]
Abandoning this PR as all of these will be fixed w/ https://github.com/IgniteUI/igniteui-angular-samples/pull/2700

**Pull request closed without merging**